make_image.sh

```diff
@@ -0,0 +1,231 @@
+#!/bin/bash
+
+set -o errtrace
+set -o nounset
+
+ROOT="$( cd $( dirname $0 ); pwd )"
+
+TOOLCHAIN_DIR_DEFAULT="/opt/grisp/grisp2-rtems-toolchain/rtems/5"
+BAREBOX_IMG_FILENAME="barebox-phytec-phycore-imx6ul-emmc-512mb.img"
+BAREBOX_RELDIR="../../barebox"
+OUTPUT_FILE="$PWD/grisp2_emmc.img.gz"
+LOOPBACK_ID="99"
+
+TOOLCHAIN_DIR=${GRISP_TOOLCHAIN:-$TOOLCHAIN_DIR_DEFAULT}
+BAREBOX_IMG=""
+ERLANG_APP_DIR="$ROOT"
+ERLANG_APP_NAME="grisp_demo"
+ERLANG_APP_VSN="0.1.0"
```

Comment:
Can we skip hardcoding these values here? This will just lead to creating images with old versions.

Reply:
So you mean to make these parameter required... YEs I can do it, I liked not having to pass them all the time until now :)

Reply:
The app name shouldn't really change though.... only the version...

Reply:
Then make them environment variables instead?

Reply:
I removed them too